_posts/2025-09-15-quarkus-a2a-multi-agent-content-creation.adoc

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: 'Multi-Language Agent Collaboration and Interoperability with A2A'
+date: 2025-09-15
+tags: ai a2a
+synopsis: Let's learn how to build a multi-agent system where agents written in different languages and with different LLM frameworks can seamlessly collaborate using A2A.
+author: fjuma
+---
+:imagesdir: /assets/images/posts/quarkus-a2a-multi-agent-content-creation
+
+Building a multi-agent system can involve using different languages to meet specific needs. The https://a2a-protocol.org/latest/[Agent2Agent (A2A) protocol] is an open standard that enables AI agents to communicate and collaborate with one another, regardless of each agent's underlying technology stack.
+
+In this post, we'll see how to create a multi-agent system, where agents written in Java, Python, and TypeScript work together to accomplish a goal: content creation. The multi-agent system uses A2A for communication between the AI agents.
+
+image::ContentCreationDiagram.png[scaledwidth=100%]
+
+== Content Creation Sample
+
+We're going to do a deep dive into the https://github.com/a2aproject/a2a-samples/tree/main/samples/python/hosts/content_creation[Content Creation] sample from the https://github.com/a2aproject/a2a-samples[a2a-samples] project.
+
+This sample showcases a content creation pipeline with a Host agent that acts as the central orchestrator,
```

Comment:
Why `Host` with an uppercase?

Reply:
"Host" is the agent name that I'm using for the central orchestrator, it's referenced in the table and in the diagrams. I'll update this to use `Host` instead to hopefully make it more clear.